CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+5.9.3 (August 2024)
+-------------------
+
+- Fix order of the directory separators when downloading an image (https://github.com/ome/omero-gateway-java/pull/90)
+- Bump org.openmicroscopy.project 5.6.0 (https://github.com/ome/omero-gateway-java/pull/69)
```

Comment:
I don't think we include that one since we have not included it in the other repositories

Reply:
👍